apps/dapp-console/app/settings/components/CoinbaseVerificationBanner.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image'
+
+import { Text } from '@eth-optimism/ui-components/src/components/ui/text'
+
+export const CoinbaseVerificationBanner = () => (
+  <div className="flex flex-row bg-[#d6e4ff] w-full p-8 rounded-xl items-center">
```

Comment:
the static `bg-[#d6e4ff]` will look a little off in dark mode, instead we can use `bg-blue-500/20` to set the opacity of the token to 20% which achieves the same visual on light mode + gives us a nice dark mode variant as well.

Reply:
sick i didn't realize you can handle opacity this way, the changes look nice 👍 thanks!